components/list/list-item-link-mixin.js

```diff
@@ -21,14 +21,27 @@ export const ListItemLinkMixin = superclass => class extends ListItemMixin(super
 			:host([action-href]:not([action-href=""])) {
 				--d2l-list-item-content-text-color: var(--d2l-color-celestine);
 			}
-			a[href] {
+			:host([_list-item-interactive-enabled]) a[href] {
 				color: unset;
 				display: block;
 				height: 100%;
 				outline: none;
 				text-decoration: none;
 				width: 100%;
 			}
+			/** clean up with flag GAUD-7495-list-interactive-content */
+			:host(:not([_list-item-interactive-enabled])) a[href] {
+				display: block;
+				height: 100%;
+				outline: none;
+				width: 100%;
+			}
+			/** clean up with flag GAUD-7495-list-interactive-content */
+			:host(:not([_list-item-interactive-enabled])[action-href]:not([action-href=""])) [slot="content"],
+			:host(:not([_list-item-interactive-enabled]):not([no-primary-action])) [slot="control-action"] ~ [slot="content"],
+			:host(:not([_list-item-interactive-enabled]):not([no-primary-action])) [slot="outside-control-action"] ~ [slot="content"] {
+				pointer-events: none;
+			}
```

Comment:
Adds back in the changes [here](https://github.com/BrightspaceUI/core/pull/5520/files#diff-4b504def2c2a3936740c0b7890ace8c73d3934afd29f1526aa1104569ffc8dca) and checks for the flag.